src/SampleSpec.ts

```diff
@@ -1,3 +1,4 @@
+import { SampleIModels } from "@itwinjs-sandbox";
```

Comment:
Extract this enum into separate file without any imports to be used here. Those imports drags in megabytes of dependencies to initial bundle size. The same applies to sampleSpec files where this interface is used

Reply:
Moved SampleIModels to its own file and changed import statements to reference the file directly.